numpy/core/src/multiarray/flagsobject.c

```diff
@@ -339,8 +362,9 @@ arrayflags_writebackifcopy_set(PyArrayFlagsObject *self, PyObject *obj)
                 "Cannot set flags on array scalars.");
         return -1;
     }
-    res = PyObject_CallMethod(self->arr, "setflags", "OOO", Py_None, Py_None,
-                              (PyObject_IsTrue(obj) ? Py_True : Py_False));
+    res = PyObject_CallMethod(self->arr, "setflags", "OOO0", Py_None, Py_None,
+                              (PyObject_IsTrue(obj) ? Py_True : Py_False),
+                              Py_None);
```

Comment:
Is this addition still necessary? Below you comment that you're not mangling `setflags` (I'm probably just confused...)

Reply:
reverted cruft left over from intermediate versions